pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -226,6 +227,12 @@ func (cr *PerconaPGCluster) ToCrunchy(ctx context.Context, postgresCluster *crun
 	annotations := make(map[string]string)
 	for k, v := range cr.Annotations {
 		switch k {
+		// pg-backup reconciler will manage this annotation
+		case naming.PGBackRestBackup:
+			v, ok := postgresCluster.Annotations[k]
+			if ok {
+				annotations[k] = v
+			}
```

Comment:
why do we explicitly need this case? don't the default case handle this as well?

Reply:
This part ignores the annotation from the `PerconaPGCluster` and sets the previous value from the `PostgresCluster`.

It was removed because of complicated logic: https://github.com/percona/percona-postgresql-operator/pull/634#discussion_r1452808451